src/Worker.Extensions.DurableTask/FunctionsDurableTaskClient.cs

```diff
@@ -76,6 +77,12 @@ public override Task<string> ScheduleNewOrchestrationInstanceAsync(
         StartOrchestrationOptions? options = null,
         CancellationToken cancellation = default)
     {
+        Activity? currActivity = Activity.Current;
+        if (options == null && options?.ParentTraceId == null && currActivity != null)
+        {
+            options = new StartOrchestrationOptions(ParentTraceId: currActivity?.Id?.ToString());
+        }
```

Comment:
I added the `ParentTraceId` information to the `StartOrchestrationOptions`, but I'm wondering if there's a better place to store this information since `StartOrchestrationOptions` can be configured by the customer and we don't necessarily need customers to have access to `ParentTraceId`.